### DISCUSSION TITLE
Establishing Modelica logo on simple-icons failed

### DISCUSSION THREAD

**Discussion by tobolar** [modelica/ModelicaStandardLibrary#4237] [Show and tell]
To whom this may concern - just for your information: https://github.com/simple-icons/simple-icons/issues/9700

**Comment by HansOlsson** [comment]
Well, similar-web ranking should be less than 500k or GitHub above 5k stars according to their guidelines.
We will need to work hard to reach that.

**Reply by tobolar** [reply]
Ok, there is a small gap to reach it. ;-)
